src/main/java/frc/robot/commands/armCommands/PositionCommand.java

```diff
@@ -0,0 +1,111 @@
+package frc.robot.commands.armCommands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.subsystems.arm.Arm;
+import frc.robot.subsystems.arm.ArmPositions;
+import frc.robot.subsystems.intake.Intake;
+
+public class PositionCommand extends SequentialCommandGroup {
+    ElbowPosition elbowCommand;
+    ShoulderPosition shoulderPosition;
+    Arm arm;
+    Intake intake;
+
+    public PositionCommand(Arm armMech, Intake intakeMech){
+        this.arm = armMech;
+        this.intake = intakeMech;
+    }
+    
+    // public SequentialCommandGroup armStowCommand(){
```

Comment:
Should all these commented out sections be uncommented?  Or deleted?

Reply:
comments have been deleted